vue2/src/components/Events.vue

```diff
@@ -1,39 +1,32 @@
+<template>
+  <div>
+    <EventCard v-for="event in events" :key="event.id" :event="event"/>
+  </div>
+</template>
+
 <script>
+import { mapState } from 'vuex';
+import EventCard from '@/components/EventCard.vue';
+
 export default {
-  props: {
-    user: {
-      type: String,
-      required: true,
-    }
-  },
-  data() {
-    return {
-      events: [], // 1
-      filters: {}, // 3
-      searchQuery: '' // 2
-    };
+  components: {
+    EventCard,
   },
   computed: {
-    filteredEvents () { // 3
+    filteredEvents() { // 3
       return [];
     },
-    eventsMatchingSearchQuery () { // 2
+    eventsMatchingSearchQuery() { // 2
       return [];
     },
+    events() {
+      return this.event.events;
+    },
+    ...mapState(['event', 'user']),
```

Comment:
이 컴포넌트에서는 요 4줄만 빼고는 사용하는 곳이 없군요. 👀

Reply:
@KunHwanAhn `computed` 안에서 22 ~ 25번 라인까지만 활용되고 있는 점에 대해서 말씀해주신거군요. 사용되지 않는 부분은 필터 및 검색 기능을 추가해서 compositionApi 사용방법과 비교하는 대상으로 만들어보려고 해요. 이번 주안에 구현예정입니다. `mapState`의 `user`는 현재 사용하는 부분이 없어서 제거해야 겠네요.
```suggestion
    events() {
      return this.event.events;
    },
    ...mapState(['event']),
```